src/RedirectionForm.php

```diff
@@ -180,11 +180,12 @@ public function render() : string
      *
      * @param $options
      *
-     * @return \Illuminate\Http\JsonResponse
+     * @return string|bool
      */
-    public function toJson($options = JSON_UNESCAPED_UNICODE) : \Illuminate\Http\JsonResponse
+    public function toJson($options = JSON_UNESCAPED_UNICODE) : mixed
```

Comment:
mixed type supported only in php 8. 
please remove the returning type, we need to support PHP 7+